anakondo.el

```diff
@@ -197,17 +202,28 @@ and the completion candidates for it at cdr for buffer.")
 
 We try to find the project root by:
 1. Trying to query `clojure-mode' for it.
-2. Trying to query projectile for it.
-3. Defaulting to the `default-directory' of the buffer otherwise.
+2. Trying to query `project' for it.
+3. Trying to query projectile for it.
+4. Defaulting to the `default-directory' of the buffer otherwise.
 
 Anaphoric macro, binds `root' implicitly."
-  `(let* ((root (or (clojure-project-dir)
-                    (projectile-project-root)
+  `(let* ((root (or (and (featurep 'clojure-mode) (clojure-project-dir))
+                    (anakondo--project-get-project-root)
+                    (and (featurep 'projectile) (projectile-project-root))
                     default-directory)))
      ,@body))
 
 ;;;; Functions
 
+(defun anakondo--project-get-project-root ()
```

Comment:
Suddenly I have a blank, can you define a function after its call-site in Emacs Lisp? In Clojure no, so now I can't remember?

Reply:
Yes, Emacs Lisp supports this. I've seen it in various packages too